tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -322,6 +327,56 @@ describe('Eventing tests', function() {
     });
   }
 
+  function jsSaveStreamAndConsumersDataSuite() {
+    context('save stream and consumer data', function() {
+      it('saving JetStream data to test stream and consumers will not be re-created by kyma upgrade', async function() {
+        if (!isEventingSinkDeployed || !testSubscriptionV1Alpha2) {
```

Comment:
why only with V1Alpha2?

Reply:
v1alpha1 is currently covered in [here](https://github.com/kyma-project/kyma/blob/dbbe74c5ee167a0e9561dd23bc5e0b7d08d20d28/tests/fast-integration/eventing-test/eventing-test.js#L327), which is based of the subscription created for last-order function. Later, when we have the new release, we can cleanup all this code.